### DISCUSSION TITLE
Weird cursor rendering when auto pilot mod in osu!lazer in replay mode and replay file was exported with osu stable

### DISCUSSION THREAD

**Discussion by timelessnesses** [ppy/osu#16799] [Q&A]
entire of this [replay](https://cdn.rukchadisa.live/r/Whistle_Python_-_Laur_-_Pristine_[Viola]_(2565-01-28)_Osu-1.osr)  
there's a cursor move up and down before it hit circle?  
and sometimes cursor being glitchy?

**Comment by timelessnesses** [comment]
beatmap https://osu.ppy.sh/beatmapsets/840447#osu/1759213

**Comment by timelessnesses** [comment]
logs: [logs](https://cdn.rukchadisa.live/r/logs_2.zip)

**Comment by frenzibyte** [comment]
Seems `OsuModAutopilot` still applies mouse input even when it is a replay, I think on cases like these we would want to [follow what was done for `OsuModRelax` already and just not apply anything](https://github.com/ppy/osu/blob/cbd563e80b3b82aecf9b84e474197d9c12993b16/osu.Game.Rulesets.Osu/Mods/OsuModRelax.cs#L45-L59)?

**Reply by peppy** [reply]
Also of note is likely https://github.com/ppy/osu/issues/9243
